catalogs/catalog-hive/src/test/java/com/datastrato/gravitino/catalog/hive/TestHiveCatalog.java

```diff
@@ -41,7 +41,7 @@ public void testListDatabases() throws TException, InterruptedException {
     metastore.hiveConf().forEach(e -> conf.put(e.getKey(), e.getValue()));
 
     try (HiveCatalogOperations ops = new HiveCatalogOperations()) {
-      ops.initialize(conf, entity.toCatalogInfo());
+      ops.initialize(conf, entity.toCatalogInfo(), new HiveCatalog());
```

Comment:
It's a bit weird to use `HiveCatalog` here, may be we can have a default test class here.

Reply:
fixed